uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -572,6 +572,18 @@ public function upgrade_1024()
     return TRUE;
   }
 
+  /**
+   * Add new custom fields for activities
+   *
+   * @return bool
+   */
+  public function upgrade_1025()
+  {
```

Comment:
Opening curly brace should be on same line as function name according to Drupal coding standards